examples/terraform-aws-rds-example/main.tf

```diff
@@ -0,0 +1,78 @@
+data "aws_vpc" "default" {
```

Comment:
Please check some of our other examples for the commenting style we use. It's much quicker to scan the code if you group resources together logically and have a comment block labeling the grouping.